airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -253,3 +253,16 @@ def _get_secret(self, path_prefix, secret_id: str) -> Optional[str]:
                 secret_id,
             )
             return None
+        except self.client.exceptions.AccessDeniedException as e:
+            self.log.debug(
+                "An error occurred (AccessDeniedException) when calling the "
+                "get_secret_value operation: %s",
+                str(e),
+            )
+            return None
+        except Exception as e:
```

Comment:
Since all the calls to the secret backends are inside a try catch , it's not necessary to add this last catch in your PR 

see -> https://github.com/apache/airflow/blob/9519bf664b1c20ebec940fc1c3daa097eca351b4/airflow/models/variable.py#L270

Reply:
Removed.  Thanks!